packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -106,6 +107,11 @@ export interface IDropdownProps extends React.Props<Dropdown> {
   required?: boolean;
 
   /**
+   * Custom properties for Dropdown's Callout.
+   */
+  calloutProperties?: ICalloutProps;

```

Comment:
@micahgodbolt , @mdahamiwal,  seems like pepple have different opinions about passing className or callout props, could we get agreement here?